hm_1/tasks/task_02_01.py

```diff
@@ -0,0 +1,72 @@
+"""
+Given a file containing text. Complete using only default collections:
+    1) Find 10 longest words consisting from largest amount of unique symbols
+    2) Find rarest symbol for document
+    3) Count every punctuation char
+    4) Count every non ascii char
+    5) Find most common non ascii char for document
+"""
+import string
+from collections import Counter
+from typing import List
+
+
+def get_data(file_path: str) -> str:
+    data = ""
+    with open(file_path, encoding="unicode_escape") as f:
+        for line in f:
+            data += line
+    return data
+
+
+def get_longest_diverse_words(file_path: str) -> List[str]:
+    data = {}
+    result = []
+    text_without_punctuation = get_data(file_path).translate(
```

Comment:
`text_without_punctuation = get_data(...`

You use the name `get_data` for function but use it to assign text. This is a little bit confusing, especially because you have the variable named data too.

Usually, it is expected that name of the function and variable which holds the result looks alike.